src/python/dxpy/scripts/dx_build_app.py

```diff
@@ -230,12 +230,12 @@ def _check_syntax(code, lang, temp_dir, enforce=True):
     """
     # This function needs the language to be explicitly set, so we can
     # generate an appropriate temp filename.
-    if lang == 'python2.7':
+    if lang == 'python2.7' or lang == 'python3':
```

Comment:
Actually, it could cover other "python3" version added down the road (including python3.5), so how about:

```suggestion
    if lang == 'python2.7' or 'python3' in lang':
```